resource-service/common/git.go

```diff
@@ -80,23 +80,23 @@ func getGitKeptnEmail() string {
 }
 
 func getAuthMethod(gitContext common_models.GitContext) (transport.AuthMethod, error) {
-	if gitContext.Credentials.GitPrivateKey != "" && strings.HasPrefix(gitContext.Credentials.RemoteURI, "ssh://") {
-		publicKey, err := ssh.NewPublicKeys("git", []byte(gitContext.Credentials.GitPrivateKey), gitContext.Credentials.GitPrivateKeyPass)
+	if gitContext.Credentials.SshAuth != nil {
```

Comment:
I would expect `*.[https|ssh].*` as a format without the `Auth` part

Reply:
then we need to change the models in go-utils :)

Reply:
we can do that in a follow up